internal/transactionsubmission/monitor/monitor_service.go

```diff
@@ -0,0 +1,123 @@
+package monitor
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stellar/go/support/log"
+
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/monitor"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/transactionsubmission/store"
+)
+
+type MonitorService struct {
```

Comment:
not sure if we should give this a different name to differentiate it from the one one in SDP's

Reply:
Maybe TSSMonitorService?